app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminClickInvoiceButtonIntoOrderActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickInvoiceButtonIntoOrderActionGroup">
```

Comment:
I would suggest renaming this action group slightly, since "Into" does not describe the purpose accurately. Something like 

"AdminClickInvoiceButtonOrderViewActionGroup"

What do you think?

Reply:
**Done**